src/WidgetBase.php

```diff
@@ -196,6 +205,14 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
       '#default_value' => $this->configuration['submit_text'],
     ];
 
+    // Allow "auto_select" setting when autoSelect is supported by widget.
```

Comment:
Currently - auto-selection only works in combination with multistep display, that's why I have added check in EntityBrowserForm.php to throw exception if such configuration is defined. In general, it could be possible to make it in combination with other selection displays -> if you take a look in dropzonejs PR, you will see that widget form is actually submitted (same as it would be with button click), in that submit action it is possible to select entities, but after that request is finished, you have to rebuild EB form. And all this functionality is about -> not to rebuild form, just relevant parts.